src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -158,8 +158,11 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
 
   handleLogsError(data: PanelData) {
     const errorResponse = data.errors?.length ? data.errors[0].message : data.error?.message;
-    let error = 'Unexpected error response. Please review your filters or try a different time range.';
+    if (errorResponse) {
+      logger.error(errorResponse);
```

Comment:
I would pass some additional context from the DataQueryError, the faro `logError` method only takes an Error class

```suggestion
      logger.error(new Error('Logs panel error'), {msg: errorResponse, status: data.errors?.[0]?.status, type: data.errors?.[0]?.type});
```